lib/ansible/modules/cloud/cloudstack/cs_vpc_offering.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, David Passante (@dpassante)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: cs_vpc_offering
+short_description: Manages vpc offerings on Apache CloudStack based clouds.
+description:
+    - Create, update, enable, disable and remove CloudStack VPC offerings.
+version_added: '2.5'
+author: "David Passante (@dpassante)"
+options:
+  name:
+    description:
+      - The name of the vpc offering
+    required: true
+  state:
+    description:
+      - State of the vpc offering.
+    choices: [ enabled, present, disabled, absent ]
+    required: false
+    default: present
+  display_text:
+    description:
+      - Display text of the vpc offerings
+    required: false
+  service_capabilities:
+    description:
+      - Desired service capabilities as part of vpc offering.
+    aliases: [ service_capability ]
+  service_offering:
+    description:
+      - The name or ID of the service offering for the VPC router appliance.
+    required: false
+  supported_services:
+    description:
+      - Services supported by the vpc offering
+    aliases: [ supported_service ]
+    required: false
+  service_providers:
+    description:
+      - provider to service mapping. If not specified, the provider for the service will be mapped to the default provider on the physical network
+    aliases: [ service_provider ]
+    required: false
+  poll_async:
+    description:
+      - Poll async jobs until job has finished.
+    default: true
+extends_documentation_fragment: cloudstack
+'''
+
+EXAMPLES = '''
+# Create a vpc offering and enable it
+- local_action:
+    module: cs_vpc_offering
+    name: "my_vpc_offering"
+    display_text: "vpc offering description"
+    state: enabled
+    supported_services: [ Dns, Dhcp ]
+    service_providers:
+      - {service: 'dns', provider: 'virtualrouter'}
+      - {service: 'dhcp', provider: 'virtualrouter'}
+
+# Remove a vpc offering
+- local_action:
+    module: cs_vpc_offering
+    name: "my_vpc_offering"
+    display_text: "vpc offering description"
+    state: absent
```

Comment:
would you mind minimalizing the example for state=absent to the things required:

~~~yaml
- name: Remove a vpc offering
  local_action:
    module: cs_vpc_offering
    name: my_vpc_offering
    state: absent
~~~